front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -128,22 +136,25 @@ export function AgentMessage({
         })(message.status);
     }
   })();
-
+  const buttons =
```

Comment:
@lasryaric this part of the code in this PR is just avoiding copy & feedback buttons on error messages, but is not related to the "buttons" interface of conversation message per se so you don't need to check that
however, if you'd like I can explain what "buttons" are and why I did them (in a former PR)